src/shared/api/urls.ts

```diff
@@ -65,6 +65,9 @@ export function getOncoKbApiUrl() {
 export function getPdbAnnotationApiUrl() {
     return 'https://cbioportal.mskcc.org/pdb-annotation';
 }
+export function getG2SApiUrl() {
+    return 'http://g2s.genomenexus.org';
```

Comment:
This one will NOT work with secure portal instances, we need to replace this one with the `https` version, or we need to use `proxy` for a temporary workaround.